inst/Upgrades.qml

```diff
@@ -0,0 +1,289 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName: 		"BinomialTest"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
+
+		ChangeRename{	from: "hypothesis";	to: "alternative"	}
+		ChangeJS
+		{
+			name:		"alternative"
+			jsFunction:	function(options)
+			{
+				switch(options["alternative"])
+				{
+					case "notEqualToTestValue":		return "twoSided";
```

Comment:
I forgot how this works. Would "less" now be the default, because it appears last?

Reply:
No, the upgrader does not affect the defaults, that's still set by the analysis qml.

I think you maybe confused it with the `default` in a `switch` statement? That could be used here if there were some radio buttons in this groups that did not change names. E.g.

radiobuttons:
- `A` -> `a`
- `B` -> `b`
- `c` -> `c` (stays the same)
- `d` -> `d` (stays the same)

could be upgraded with 
```js
switch(options["radiobuttons"])
{
	case "A":	return "a";
	case "B":	return "b";
	default:	return options["radiobuttons"];
)
```